install.sh

```diff
@@ -7,9 +7,9 @@ nvm_has() {
   return $?
 }
 
-if [ -z "$NVM_DIR" ]; then
-  NVM_DIR="$HOME/.nvm"
-fi
+nvm_install_dir() {
+  echo ${NVM_DIR:-"$HOME/.nvm"}
```

Comment:
1.5 years later, I now know this to be fully portable and compatible :-) thanks for introducing it.